packages/marko-web-theme-monorail/components/gam/components/define-display-ad.marko

```diff
@@ -0,0 +1,102 @@
+import { BEM, warn } from "@parameter1/base-cms-utils";
```

Comment:
add an `@todo` here: Merge this component into marko-web-gam's define-display-ad. Bring over the utility function and add the referrer and position targeting.

This should have been done previously instead of making a custom component for it, but will let go for now.